app/assets/stylesheets/top-bar.scss

```diff
@@ -2,6 +2,12 @@
 @import 'variables';
 @import 'mixins';
 
+body.default-navbar-config {
+  .article .body [id] {
+    scroll-margin-top: var(--header-height);
+  }
+}
+
```

Comment:
Lovely! Can I have one more request to actually put it in [this file](https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/article-show.scss) instead of topbar? It is more article-related issue rather than top bar :). But overall, it's great!

Reply:
Done! See most recent commit~